.github/ISSUE_TEMPLATE/2-bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: 🐞 Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+### Before You Start
+This form is only for submitting bug reports.
+
+### Describe the bug
+Please provide us with a clear and concise description of what the bug is.
+
+### Expected behavior
+A clear and concise description of what you expected to happen.
+
+### How to reproduce
+It would be awesome if you could provide us a small reproduction online using stackblitz. You can replace `typescript-vitest` with another template name that is matching your case from https://github.com/vuejs/create-vue-templates.
+- https://stackblitz.com/github/vuejs/create-vue-templates/tree/main/typescript-vitest
+
+Steps to reproduce the behavior:
+1. Go to '....'
+2. Do this '....'
+3. Do that '....'
+4. Result
+
+### Screenshots
+Add screenshots to help explain your problem. (If applicable)
```

Comment:
I would remove the screenshot part, as it is often easier to read the error messages in plain text